### PR TITLE
Feature/api catalogue servicelist

### PR DESCRIPTION
**Pull request by saevar43**
# Feature/api-catalogue-servicelist

## What

Implement and integrate a service list component into the web. Implement graphql queries to populate the service list.

## Why

The service list component is intended to display a list of web services.

## Screenshots / Gifs

servicelist

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/ApiCatalogue/ApiCatalogue.tsx [hunk 1]
```diff
@@ -1,30 +1,209 @@
...
+            </SidebarLayout>
+          }
+        />
+      }
+    />
+  )
 }
 
 ApiCatalogue.getInitialProps = async ({ apolloClient, locale, query }) => {
+  console.log(locale)
```
Comment (darrikonn) [inline apps/web/screens/ApiCatalogue/ApiCatalogue.tsx:166]: console

Comment (saevar43) [reply apps/web/screens/ApiCatalogue/ApiCatalogue.tsx:166]: ah yes, must have slipped through. Will remove.

File: apps/web/screens/ApiCatalogue/ApiCatalogue.tsx [hunk 2]
```diff
@@ -1,30 +1,209 @@
...
+                  />
+                  {data?.getApiCatalogue?.pageInfo?.nextCursor != null && (
+                    <Box display="flex" justifyContent="center">
+                      <Button
+                        colorScheme="default"
+                        iconType="filled"
+                        onClick={() => onLoadMore()}
+                        size="default"
+                        type="button"
+                        variant="ghost"
```
Comment (barabrian) [inline apps/web/screens/ApiCatalogue/ApiCatalogue.tsx:145]: no need to define default values
```suggestion
                      <Button
                        onClick={() => onLoadMore()}
                        variant="ghost"
```

File: apps/web/screens/ApiCatalogue/ApiCatalogue.tsx (hunk 1 above)
Comment (barabrian) [inline apps/web/screens/ApiCatalogue/ApiCatalogue.tsx:166]: is this needed?

File: apps/web/screens/ApiCatalogue/ApiCatalogue.tsx [hunk 3]
```diff
@@ -1,30 +1,209 @@
...
 
   if (disablePage === 'true') {
     throw new CustomNextError(404, 'Not found')
   }
 
-  return <h1>{title}</h1>
+  const sn = useNamespace(staticContent)
+
+  const onLoadMore = () => {
+    if (data?.getApiCatalogue.pageInfo?.nextCursor == null) {
```
[the remark this answers is not in the document]
Comment (saevar43) [reply apps/web/screens/ApiCatalogue/ApiCatalogue.tsx]: fixed

File: apps/web/screens/ApiCatalogue/ApiCatalogue.tsx (hunk 2 above)
Comment (saevar43) [reply apps/web/screens/ApiCatalogue/ApiCatalogue.tsx]: fixed

**Review by barabrian — APPROVED** [review]
LGTM

**Review by saevarma — APPROVED** [review]
LGTM

**Review by barabrian — APPROVED** [review]

**Review by darrikonn — APPROVED** [review]

**Pull request merged**